### PR TITLE
docs / add docs for client commands

### PR DESCRIPTION
**Pull request by yzhang1994**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

### CODE REVIEW

File: documentation/docs/cheatsheets/client.md [hunk 1]
```diff
@@ -22,5 +22,24 @@
...
+| `CTRL + S` | Status | Show bot status
+| `CTRL + F` | *Search / Hide Search | Toggle search in log pane
+| `CTRL + A` | Select All | Select all text in input pane [Used for text edit in Input pane only] 
+| `CTRL + Z` | Undo | Undo action in input pane [Used for text edit in Input pane only] 
+| `Single CTRL + C` | Copy | Copy text [Used for text edit in Input pane only] 
+| `CTRL + V` | Paste | Paste text [Used for text edit in Input pane only] 
+
+***Note about search:*** 
+1. Press `CTRL + F` to trigger display the search field
+2. Enter your search keyword (Not case sensitive)
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: lower case "Not"

File: documentation/docs/cheatsheets/client.md [hunk 2]
```diff
@@ -22,5 +22,24 @@
...
+3. Hit `Enter` to jump to the next matching keyword (Incremental search)
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: Lower case "Incremental"

File: documentation/docs/cheatsheets/client.md [hunk 3]
```diff
@@ -22,5 +22,24 @@
...
+| `bounty --restore-id` | If you lost your configuration file, this is the command to restore it.
 | `bounty --status` | See your accumulated rewards
 | `bounty --terms` | See the terms & conditions
+
+## Keyboard shortcuts
+| Keyboard Combo | Command | Description |
+|-------- | ----------- | ----------- |
+| `Double CTRL + C` | Exit | Press `CTRL + C` twice to exit the bot
+| `CTRL + S` | Status | Show bot status
+| `CTRL + F` | *Search / Hide Search | Toggle search in log pane
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: is the * in *Search intentional?

File: documentation/docs/cheatsheets/client.md [hunk 4]
```diff
@@ -22,5 +22,24 @@
 | Command | Description |
 |-------- | ----------- |
 | `bounty --register` | Register to participate in for liquidity bounties
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: Can you add a period at the end of this line for consistency?

File: documentation/docs/cheatsheets/client.md [hunk 5]
```diff
@@ -22,5 +22,24 @@
...
+| `bounty --list` | See a list of active bounties.
+| `bounty --restore-id` | If you lost your configuration file, this is the command to restore it.
 | `bounty --status` | See your accumulated rewards
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: ditto

File: documentation/docs/cheatsheets/client.md [hunk 6]
```diff
@@ -22,5 +22,24 @@
...
 | `bounty --terms` | See the terms & conditions
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: ditto

File: documentation/docs/operation/client.md [hunk 7]
```diff
@@ -57,5 +57,24 @@ The CLI is divided into three panes:
 | Command | Description |
 |-------- | ----------- |
 | `bounty --register` | Register to participate in for liquidity bounties
+| `bounty --list` | See a list of active bounties.
```
Comment (carlolm) [inline documentation/docs/operation/client.md]: Same comments as above.

File: documentation/docs/cheatsheets/client.md [hunk 8]
```diff
@@ -22,5 +22,24 @@
...
+|-------- | ----------- | ----------- |
+| `Double CTRL + C` | Exit | Press `CTRL + C` twice to exit the bot
+| `CTRL + S` | Status | Show bot status
+| `CTRL + F` | *Search / Hide Search | Toggle search in log pane
+| `CTRL + A` | Select All | Select all text in input pane [Used for text edit in Input pane only] 
+| `CTRL + Z` | Undo | Undo action in input pane [Used for text edit in Input pane only] 
+| `Single CTRL + C` | Copy | Copy text [Used for text edit in Input pane only] 
+| `CTRL + V` | Paste | Paste text [Used for text edit in Input pane only] 
+
+***Note about search:***
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: Should there be a description of what this does?  Searches the log pane.  Kind of hidden in keyboard shortcuts.

File: documentation/docs/cheatsheets/client.md [hunk 9]
```diff
@@ -22,5 +22,24 @@
...
 | `bounty --status` | See your accumulated rewards
 | `bounty --terms` | See the terms & conditions
+
+## Keyboard shortcuts
+| Keyboard Combo | Command | Description |
+|-------- | ----------- | ----------- |
+| `Double CTRL + C` | Exit | Press `CTRL + C` twice to exit the bot
+| `CTRL + S` | Status | Show bot status
+| `CTRL + F` | *Search / Hide Search | Toggle search in log pane
+| `CTRL + A` | Select All | Select all text in input pane [Used for text edit in Input pane only]
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: lower case "Used"

File: documentation/docs/cheatsheets/client.md [hunk 10]
```diff
@@ -22,5 +22,24 @@
...
 | `bounty --terms` | See the terms & conditions
+
+## Keyboard shortcuts
+| Keyboard Combo | Command | Description |
+|-------- | ----------- | ----------- |
+| `Double CTRL + C` | Exit | Press `CTRL + C` twice to exit the bot
+| `CTRL + S` | Status | Show bot status
+| `CTRL + F` | *Search / Hide Search | Toggle search in log pane
+| `CTRL + A` | Select All | Select all text in input pane [Used for text edit in Input pane only] 
+| `CTRL + Z` | Undo | Undo action in input pane [Used for text edit in Input pane only]
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: ditto

File: documentation/docs/cheatsheets/client.md [hunk 11]
```diff
@@ -22,5 +22,24 @@
...
+
+## Keyboard shortcuts
+| Keyboard Combo | Command | Description |
+|-------- | ----------- | ----------- |
+| `Double CTRL + C` | Exit | Press `CTRL + C` twice to exit the bot
+| `CTRL + S` | Status | Show bot status
+| `CTRL + F` | *Search / Hide Search | Toggle search in log pane
+| `CTRL + A` | Select All | Select all text in input pane [Used for text edit in Input pane only] 
+| `CTRL + Z` | Undo | Undo action in input pane [Used for text edit in Input pane only] 
+| `Single CTRL + C` | Copy | Copy text [Used for text edit in Input pane only]
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: ditto

File: documentation/docs/cheatsheets/client.md [hunk 12]
```diff
@@ -22,5 +22,24 @@
...
+## Keyboard shortcuts
+| Keyboard Combo | Command | Description |
+|-------- | ----------- | ----------- |
+| `Double CTRL + C` | Exit | Press `CTRL + C` twice to exit the bot
+| `CTRL + S` | Status | Show bot status
+| `CTRL + F` | *Search / Hide Search | Toggle search in log pane
+| `CTRL + A` | Select All | Select all text in input pane [Used for text edit in Input pane only] 
+| `CTRL + Z` | Undo | Undo action in input pane [Used for text edit in Input pane only] 
+| `Single CTRL + C` | Copy | Copy text [Used for text edit in Input pane only] 
+| `CTRL + V` | Paste | Paste text [Used for text edit in Input pane only]
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: ditto

File: documentation/docs/cheatsheets/client.md [hunk 13]
```diff
@@ -22,5 +22,24 @@
 | Command | Description |
 |-------- | ----------- |
 | `bounty --register` | Register to participate in for liquidity bounties
+| `bounty --list` | See a list of active bounties.
+| `bounty --restore-id` | If you lost your configuration file, this is the command to restore it.
 | `bounty --status` | See your accumulated rewards
 | `bounty --terms` | See the terms & conditions
+
+## Keyboard shortcuts
```
Comment (carlolm) [inline documentation/docs/cheatsheets/client.md]: Add periods to the end of sentences throughout this table.

File: documentation/docs/cheatsheets/client.md (hunk 3 above)
Comment (yzhang1994) [reply documentation/docs/cheatsheets/client.md]: Yea it means more info down below

**Pull request merged**